OneBlink.SDK/models/CondtionallyShowPredicate.cs

```diff
@@ -38,5 +38,9 @@ public string @operator
         {
             get; set;
         }
+        public string compareWith
+        {
+            get; set;
+        }

```

Comment:
Do we need to change the value type to something else to cater for a string or number?

Reply:
ah fuck

Reply:
Looks like it needs to be `object`